src/test/resources/application-test.yml

```diff
@@ -15,4 +15,10 @@ api:
     url: http://will-be-replaced-by-wireMockServer-bean
 
 dynamodb:
-  print-details-table-name: print-details
\ No newline at end of file
+  print-details-table-name: print-details
+  scheduler-lock-table-name: scheduler-locks
+
+jobs:
+    print-requests:
+        name: BatchPrintRequests
+        cron: 0 0 */1 * * *
```

Comment:
Could you put a comment after/above the cron expression explaining what it evaluates to please?
Also (because I didn't know what the cron evaluates to) I had to look it up - I went to https://crontab.guru (my usual goto 😁 ) and it wasn't happy with the cron because it has 6 elements. Do we need a 5 or 6 element cron expression here? What format is Spring happy with? 

https://www.baeldung.com/cron-expressions (gives examples of 5 elements)
https://www.baeldung.com/cron-syntax-linux-vs-spring (describes how spring support a 6th element for some reason that I didnt read properly 🤣 )
https://spring.io/blog/2020/11/10/new-in-spring-5-3-improved-cron-expressions (suggests 6 elements?)

Reply:
> Do we need a 5 or 6 element cron expression here? What format is Spring happy with?

Spring uses 6 elements, not the UNIX 5 elements